packages/flutter/lib/src/material/slider_theme.dart

```diff
@@ -449,19 +498,23 @@ class SliderThemeData extends Diagnosticable {
       return false;
     }
     final SliderThemeData otherData = other;
-    return otherData.activeTrackColor == activeTrackColor &&
+    return otherData.trackHeight == trackHeight &&
```

Comment:
I know it wasn't like this before, so no pressure, but I think these bool expressions should all be aligned.

https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#align-expressions

like:
```
 return otherData.trackHeight == trackHeight &&
        otherData.activeTrackColor == activeTrackColor &&
        otherData.inactiveTrackColor == inactiveTrackColor &&
...
```

Reply:
Yes, they should.  A good trick for this is to put the "&& " at the beginning of the line: 

```dart
 return otherData.trackHeight == trackHeight
     && otherData.activeTrackColor == activeTrackColor
     && otherData.inactiveTrackColor == inactiveTrackColor
     && ...
```

That way it's a more consistent 4-space indent.

Reply:
Done